src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -2007,16 +2007,16 @@ If such a class is used as a base class and if the deriving class defines a dest
     <value>Invalid constraint type. A type used as a constraint must be an interface, a non-sealed class or a type parameter.</value>
   </data>
   <data name="ERR_InstanceMemberInStaticClass" xml:space="preserve">
-    <value>'{0}': cannot declare instance members in a static class</value>
+    <value>'{0}': cannot declare instance members in a static type</value>
```

Comment:
Do we support `static record`? If not, consider reverting this.

Reply:
We don't support `static record`, but unfortunately we display many cascading diagnostics in those cases, where the original message would be confusing. See `StaticRecordWithConstructorAndDestructor` for example.

Reply:
> We don't support static record, but unfortunately we display many cascading diagnostics in those cases, where the original message would be confusing.

Could we report an error for `static record`? See [https://github.com/dotnet/roslyn/issues/46450](https://github.com/dotnet/roslyn/issues/46450).

---
In reply to: [463333098](https://github.com/dotnet/roslyn/pull/46341#discussion_r463333098) [](ancestors = 463333098)